assets/prototype/domain/eventEditor/containers/mutations/useDeleteDateMutation.ts

```diff
@@ -5,7 +5,12 @@ import { GET_TICKETS } from '../queries/tickets';
 import useToaster from '../../../../infrastructure/services/toaster/useToaster';
 import useRelations from '../../../../infrastructure/services/relations/useRelations';
 
-const useDeleteDatetimeMutation = ({ eventId, id }) => {
+type useDeleteDatetimeMutationProps = {
+	eventId: string;
```

Comment:
did you pull latest `EDTR/master` before cutting this branch?
Pretty sure this has changed (or it should have) from `eventId` to `dbid`!
Can you verify that first plz?

If this **does** have the latest `EDTR/master` in it, then this looks like it got missed by Manzoor in #1956, so maybe you could update that in this PR?

Reply:
@manzoorwanijk that `eventId` is picked up from the `EventEditor` which is pulling that from `window.eeEditorEventData` in the root `index.js`.

Reply:
Checked latest master, still got that there:
![image](https://user-images.githubusercontent.com/1477453/70185906-670d2480-16f3-11ea-87ef-36205833b217.png)

Reply:
I think my diagnosis may be completely off and that will have to remain as `eventId` since in this case it is actually the relation ID and NOT the `dbid` like I was thinking

Reply:
`dbId` is not for mutation inputs.